ui/assets/css/_colors.scss

```diff
@@ -0,0 +1,53 @@
+// USWDS colors
+// https://github.com/18F/web-design-standards/blob/develop/src/stylesheets/core/_variables.scss
+
+$color-primary:              #0071bc !default;
+$color-primary-darker:       #205493 !default;
+$color-primary-darkest:      #112e51 !default;
+
+$color-primary-alt:          #02bfe7 !default;
+$color-primary-alt-dark:     #00a6d2 !default;
+$color-primary-alt-darkest:  #046b99 !default;
+$color-primary-alt-light:    #9bdaf1 !default; // lighten($color-primary-alt, 60%)
```

Comment:
FYI the comments that describe how the colors were generated here aren't actually true anymore. I've been meaning to submit a PR to the WDS to get the removed but keep forgetting 

![screen shot 2017-03-22 at 2 45 59 pm](https://cloud.githubusercontent.com/assets/776987/24223987/cfbf6498-0f26-11e7-9d49-d1f5229bf4b5.png)